lib/accessory.js

```diff
@@ -9,34 +9,44 @@ let Service, Characteristic;
 
 class Accessory {
 
-  constructor (raw, overrides) {
+  constructor (raw, overrides, log) {
     assertString(raw.powerstrip_id, '`raw.powerstrip_id` must be set');
     assertString(raw.name, '`raw.name` must be set');
     assertString(raw.device_manufacturer, '`raw.device_manufacturer` must be set');
     assertString(raw.model_name, '`raw.model_name` must be set');
     assertString(raw.serial, '`raw.serial` must be set');
     assertArray(raw.outlets, '`raw.outlets` must be an array');
     assertArray(overrides, "`overrides` must be an array");
-
+	
+	this.log = log;
     this.id = raw.powerstrip_id;
     this.name = raw.name;
     this.manufacturer = raw.device_manufacturer;
     this.model = raw.model_name;
     this.serialNumber = raw.serial;
+	this.log("Strip " + this.name + " ('" + this.model + " - S/N '" + this.serialNumber + "') initialized.");
```

Comment:
Sure, I'm learning this extra JS syntax stuff that seems to be specific to NodeJS (I assume it is?) or something else like CoffeeScript processing?